composer.json

```diff
@@ -6,24 +6,27 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
+        "drupal/core": "8.6.x-dev",
```

Comment:
It's ok to have `"drupal/core": "~8.6",` since we have `minimum-stability: dev` composer will download 8.6.x, if no stable release is found.

Reply:
:+1:  I was curious about how Composer would handle this situation, that is good to know!